migration/001-utils.sql

```diff
@@ -0,0 +1,50 @@
+ --perms for schema will be addressed later;
```

Comment:
I think there is some confusion. These are the preinstall files not the migration files. Migration files are under 
`promscale/pkg/migrations/sql/versions/dev/**` By the way, I don't think we need to port the current migration files over. But we should have some files that test the migration infrastructure.